cosmos/dbt/graph.py

```diff
@@ -324,7 +327,9 @@ def load_from_dbt_manifest(self) -> None:
 
             for unique_id, node_dict in manifest.get("nodes", {}).items():
                 node = DbtNode(
-                    name=node_dict["name"],
+                    name=node_dict["alias"]
```

Comment:
Are aliases available in older versions of `dbt`, or would it make sense to use perhaps isolate the logic of naming a `DbtNode` in a method and use `node_dict.get("alias")`?

Reply:
`alias` is required attributes of 2 resources `model` and `seed` from dbt v1.0.
Refer: [Manifest version 4](https://schemas.getdbt.com/dbt/manifest/v4/index.html#nodes_additionalProperties_oneOf_i12_alias) and [Manifest Json file document](https://docs.getdbt.com/reference/artifacts/manifest-json).